DiscordBot.Commands/VoiceControlCommands/QueueCommand.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Discord;
+using Discord.WebSocket;
+using DiscordBot.Commands.Core;
+using DiscordBot.Commands.Core.CommandAttributes;
+using DiscordBot.Core.Voice;
+using Microsoft.Extensions.Logging;
+
+namespace DiscordBot.Commands.VoiceControlCommands
+{
+    [Name("queue")]
+    [Description("Prints content of queue")]
+    public class QueueCommand : Command
+    {
+        private readonly IAudioQueueManager _audioQueueManager;
+        private readonly ILogger<QueueCommand> _logger;
+
+        public QueueCommand(IAudioQueueManager audioQueueManager, ILogger<QueueCommand> logger)
+        {
+            _audioQueueManager = audioQueueManager;
+            _logger = logger;
+        }
+
+        public override async Task ExecuteAsync(SocketSlashCommand command)
+        {
+            await command.DeferAsync();
+
+            var guildId = command.GuildId.Value;
+            var queue = _audioQueueManager.GetQueue(guildId);
+
+            if(queue is null || queue.Count == 0)
+            {
+                await command.ModifyOriginalResponseAsync(m => { m.Embed = GetEmptyQueueEmbed(); });
```

Comment:
I'm slightly afraid of a race condition here. If retrieving the queue value will be faster than Discord's API processing the defer response this might cause an issue in the future. What do you think @Niewidzialny84?

Reply:
Race contition is impossible, sending is just slower, so i will change this based on this.